lib/rubocop/formatter/worst_offenders_formatter.rb

```diff
@@ -36,14 +36,17 @@ def finished(_inspected_files)
       def report_summary(offense_counts)
         per_file_counts = ordered_offense_counts(offense_counts)
         total_count = total_offense_count(offense_counts)
+        file_count = per_file_counts.size
 
         output.puts
 
+        count_width = total_count.to_s.length + 2
         per_file_counts.each do |file_name, count|
-          output.puts "#{count.to_s.ljust(total_count.to_s.length + 2)}#{file_name}\n"
+          output.puts "#{count.to_s.ljust(count_width)}#{file_name}\n"
         end
+
         output.puts '--'
-        output.puts "#{total_count}  Total"
+        output.puts "#{total_count}  Total offenses in #{file_count} offending files"
```

Comment:
I think the word "offending" is redundant here, given the context.

Reply:
I had written it initially without "offending", but it felt a bit ambiguous: was this the total number of processed files, or the number of files with offenses?

I'm not worried too much, though, so I'm ok either way. What do you think?

Reply:
I think here it should be quiet clear from the context, so probably less is more. Even the word "offenses" is kind of redundant IMO.

Reply:
I updated it to show `3  Total in 2 files`.